setup.py

```diff
@@ -14,11 +14,15 @@
       install_requires=[
           'kafka-python',
           'singer-python==5.2.0',
-          'requests==2.20.0',
```

Comment:
we don't need _requests_ anymore?

Reply:
I hope so, everything that's required embedded in python-kafka